modules/currency-l0/src/main/scala/org/tessellation/currency/l0/CurrencyL0App.scala

```diff
@@ -58,8 +60,11 @@ abstract class CurrencyL0App(
 
     for {
       queues <- Queues.make[IO](sdkQueues).asResource
-      p2pClient = P2PClient.make[IO](sdkP2PClient, sdkResources.client, sdkServices.session)
       storages <- Storages.make[IO](sdkStorages, cfg.snapshot, method.globalL0Peer).asResource
+      selectedL0Peer <- storages.globalL0Cluster.getPeers.map { l0Peers =>
+        Random.shuffle(l0Peers.toNonEmptyList.toList).head
```

Comment:
Is there a function "getRandomPeer"? I've seen one, but I'm not sure that it's in this service. Can you verify? Also, IMO we should be consistent and it should use PeerSelect with random strategy or something like that 🤔

Reply:
Moved the peer selection logic into `RandomPeerSelect` which is passed all the `L0Peer`s.